<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>xxx.xxx.xxx.xxx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListarClientes">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar clientes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDesconectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Porta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPorta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>56</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Clientes conectados</string>
    </property>
    <widget class="QListWidget" name="listClientes">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>311</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Valores recebidos</string>
    </property>
    <widget class="Senoide" name="grafico" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Senoide</class>
   <extends>QWidget</extends>
   <header>senoide.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
